src/components/Navbar.js

```diff
@@ -84,8 +84,8 @@ const Navbar = class extends React.Component {
             <Link className="navbar-item" to="/create-project">
               Create a Project
             </Link>
-            <Link className="navbar-item" to="/projects">
-              View All Projects
+            <Link className="navbar-item" to="/browse">
```

Comment:
Huh. Can we just have this cool new functionality live at `/projects`. I don't love the `/browse` URI... You can delete the old projects.js page - we have it in git history if we need it...